.gitlab-ci.yml

```diff
@@ -386,16 +390,20 @@ publish:s3:mender-monitor:automatic:
     - *publish_helper_functions
   script:
     - echo "Publishing mender-gateway version ${MENDER_GATEWAY_VERSION} to s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/${MENDER_GATEWAY_VERSION}/"
-    - aws s3 cp output/commercial/mender-gateway*.deb s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/${MENDER_GATEWAY_VERSION}/
+    - aws s3 cp output/commercial/ s3://${S3_BUCKET_NAME_PRIVATE}/${S3_BUCKET_SUBPATH_GATEWAY}/${MENDER_GATEWAY_VERSION}/ --recursive --exclude "*" --include "mender-gateway*.deb"
```

Comment:
How this differs from the previous command?

Reply:
The awscli command is not too happy about multiple packages it seems:

```
cp
          <LocalPath> <S3Uri> or <S3Uri> <LocalPath> or <S3Uri> <S3Uri>
```

Reply:
Thanks :+1: Rings a bell, actually.